src/Application/Checker/ProductVariantLowestPriceDisplayChecker.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\PriceHistoryPlugin\Application\Checker;
+
+use Sylius\Component\Core\Model\ProductInterface;
+use Sylius\Component\Core\Model\ProductVariantInterface;
+use Sylius\Component\Core\Model\TaxonInterface;
+use Sylius\PriceHistoryPlugin\Domain\Model\ChannelInterface;
+use Webmozart\Assert\Assert;
+
+final class ProductVariantLowestPriceDisplayChecker implements ProductVariantLowestPriceDisplayCheckerInterface
+{
+    public function displayLowestPrice(ProductVariantInterface $productVariant, array $context): bool
```

Comment:
This method name is a bit misleading. At first glance, IMO the name fits the void function rather than bool
```suggestion
    public function shouldDisplayLowestPrice(ProductVariantInterface $productVariant, array $context): bool
```
```suggestion
    public function isLowestPriceDisplayable(ProductVariantInterface $productVariant, array $context): bool
```
Or similar, wdyt?